mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde

```diff
@@ -1,11 +1,11 @@
 """
-Wave Gradient 
-by Ira Greenberg.    
+Wave Gradient
+by Ira Greenberg.
 
 Generate a gradient along a sin() wave.
 """
 amplitude = 30
-fillGap = 2.5
+fillGap = 2
```

Comment:
Probably should make it 

fillGap = 3

to have the range() loop below iterate over 1,2 instead of just 1. Visually the output is the same.